src/components/common/FlexBox.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div class="flex-grid center-wrapper">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'FlexGrid'
+}
+</script>
+
+<style scoped>
+.flex-grid {
```

Comment:
Let's think about renaming this since grid and flex are actually mutually exclusive display modes, you might want to think about the semantics of how you name elements such as this. ".flex-container," perhaps.